test/functional/api/cas/cli.py

```diff
@@ -199,7 +199,7 @@ def set_param_cutoff_cmd(cache_id: str, core_id: str = None, threshold: str = No
     if threshold is not None:
         add_params += (" -t " if shortcut else " --threshold ") + str(threshold)
     if policy is not None:
-        add_params += (" -p " if shortcut else " --policy ") + policy.name
+        add_params += (" -p " if shortcut else " --policy ") + str(policy)
```

Comment:
This won't work since seq_cutoff enum does not have any __str__ defined

Reply:
removed